package.json

```diff
@@ -19,8 +19,9 @@
     "dist",
     "i18n",
     "images",
-    "materials/*.mod.css",
-    "materials/**/*.mod.css",
+    "materials/materials.css",
```

Comment:
maybe should just call them index.css and index.js?

Reply:
How about `materials/imports.css`?

Reply:
I'm also fine with the `index`, I don't mind much as long as it's proper documented